airflow/api/common/experimental/get_lineage.py

```diff
@@ -49,4 +49,4 @@ def get_lineage(
     for meta in outlets:
         lineage[meta.task_id]["outlets"] = meta.value
 
-    return {"task_ids": {k: v for k, v in lineage.items()}}
+    return {"task_ids": lineage.copy()}
```

Comment:
```suggestion
    return {"task_ids": dict(lineage)}
```

Otherwise you've got a copy of `defaultdict` rather than `dict`

Reply:
Don’t think that’d be problematic though?

Reply:
I agree with @Taragolis, I should avoid any change in the values in this refactor PR